src/darwin/Framework/CHIP/MTROTAProviderDelegate.h

```diff
@@ -85,12 +81,13 @@ typedef void (^MTRApplyUpdateRequestCompletionHandlerDeprecated)(
                                    params:(MTROTASoftwareUpdateProviderClusterApplyUpdateRequestParams *)params
                                completion:(MTRApplyUpdateRequestCompletionHandler)completion
     API_AVAILABLE(ios(16.4), macos(13.3), watchos(9.4), tvos(16.4));
+
 - (void)handleApplyUpdateRequestForNodeID:(NSNumber *)nodeID
                                controller:(MTRDeviceController *)controller
                                    params:(MTROtaSoftwareUpdateProviderClusterApplyUpdateRequestParams *)params
-                        completionHandler:(MTRApplyUpdateRequestCompletionHandlerDeprecated)completionHandler
-    API_DEPRECATED(
-        "Please use the selector ending in completion:", ios(16.1, 16.4), macos(13.0, 13.3), watchos(9.1, 9.4), tvos(16.1, 16.4));
+                        completionHandler:(MTRApplyUpdateRequestCompletionHandler)completionHandler
```

Comment:
Again, the type of the completion is being changed here and probably shouldn't be.